### PR TITLE
Add product page layout

### PR DESCRIPTION
**Pull request by AnyaLyaLya**
https://fe-feb23-code-hunters.github.io/fe_phone-catalog/#/phones/0

### CODE REVIEW

File: src/Pages/ProductDetails/ProductDetails.tsx [hunk 1]
```diff
@@ -1,14 +1,56 @@
...
+    grid__item: gridItem,
+    grid__item__mobile_1_2: gridMobileFullSize,
+    grid__item__tablet_1_6: gridTabletFullSize,
+    grid__item__desktop_1_8: gridDesktopFullSize,
+    grid__item__desktop_1_4: gridDesktopPhotoBlock,
+    grid__item__desktop_5_8: gridDesktopInfoBlock,
+    grid__item__tablet_1_4: gridTabletPhotoBlock,
+    grid__item__tablet_5_8: gridTabletInfoBlock,
+    grid__item__desktop_1_4: gridDesktopAboutBlock,
+    grid__item__desktop_5_8: gridDesktopTechSpecsBlock,
```
Comment (Hamuud) [inline src/Pages/ProductDetails/ProductDetails.tsx:42]: in our website we have 24 grid column, so: 
```suggestion
    grid__item__mobile_1_4: gridMobileFullSize,
    grid__item__tablet_1_12: gridTabletFullSize,
    grid__item__desktop_1_24: gridDesktopFullSize,
    grid__item__desktop_1_12: gridDesktopPhotoBlock,
    grid__item__desktop_13_24: gridDesktopInfoBlock,
    grid__item__tablet_1_7: gridTabletPhotoBlock,
    grid__item__tablet_8_12: gridTabletInfoBlock,
    grid__item__desktop_1_12: gridDesktopAboutBlock,
    grid__item__desktop_13_24: gridDesktopTechSpecsBlock,
```

File: src/Pages/ProductDetails/productDetails.module.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+  @media (max-width: 767px) {
+    justify-content: center;
+    grid-template-columns: repeat(8, 1fr);
+  }
+
+  @media (max-width: 639px) {
+    justify-content: center;
+    grid-template-columns: repeat(1, 1fr);
+  }
```
Comment (Hamuud) [inline src/Pages/ProductDetails/productDetails.module.scss:88]: For all this @media request we have grid.scss file, he is src/styles/grids.scss

File: src/Pages/ProductDetails/productDetails.module.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  grid-template-columns: repeat(8, 1fr);
+  margin: 0 auto;
+  padding: 24px;
+  max-width: 1200px;
+  gap: 64px;
+
+  @media (max-width: 1199px) {
+    justify-content: center;
+    grid-template-columns: 1fr;
+  }
```
Comment (Hamuud) [inline src/Pages/ProductDetails/productDetails.module.scss:102]: same to previous comment

File: src/Pages/ProductDetails/ProductDetails.tsx [hunk 4]
```diff
@@ -27,7 +69,138 @@ const ProductDetails: React.FC = () => {
...
+            gridDesktopTechSpecsBlock,
+            template,
+          )}
+        >
+          <h1 className={cn(text)}>Tech specs</h1>
+        </div>
+      </div>
+
+      <div className={cn(container, sectionMargin, template)}>
+        <h1 className={cn(title)}>You may also like</h1>
```
Comment (Hamuud) [inline src/Pages/ProductDetails/ProductDetails.tsx:202]: this is h2 title according to figma

**Review by Hamuud — CHANGES_REQUESTED** [review]
Well done, but pay attention to comment

**Review by prokivskaaa — CHANGES_REQUESTED** [review]
Change .env file link to REACT_APP_API_PATH=https://be-phone-catalog.onrender.com
And redeploy

And pull the latest changes from main

File: src/Pages/Catalog/Catalog.tsx [hunk 5]
```diff
@@ -166,6 +167,19 @@ const Catalog: React.FC = () => {
         </div>
       </div>
       <div className={cn(catalog)}>
+        <Link to="/phones/0" className={link}>
```
Comment (prokivskaaa) [inline src/Pages/Catalog/Catalog.tsx:170]: Remove this

File: src/Pages/Catalog/catalog.module.scss [hunk 6]
```diff
@@ -59,6 +59,10 @@
   }
 }
 
+.link {
```
Comment (prokivskaaa) [inline src/Pages/Catalog/catalog.module.scss:62]: Remove

**Comment by AnyaLyaLya** [comment]
image

**Review by Hamuud — APPROVED** [review]
Good work, thanks for your contribution to the project

**Review by prokivskaaa — APPROVED** [review]

**Pull request merged**